SimPEG/EM/NSEM/RxNSEM.py

```diff
@@ -66,14 +66,14 @@ def _locs_e(self):
         if self.locs.ndim == 3:
             loc = self.locs[:, :, 0]
         else:
```

Comment:
I assume I should make this a condition that if locations are provided for E and B instead making it a default to else?